submit_for_testing.py

```diff
@@ -290,12 +307,25 @@ def main():
     context.update(
         {"device_type": os.path.join(args.testplan_device_path, args.device_type)}
     )
-    test_plan_path = os.path.join(args.testplan_path, args.test_plan)
-    for test in _get_test_plan_list(test_plan_path):
+    test_list = []
+    if args.test_plan:
+        test_plan_path = os.path.join(args.testplan_path, args.test_plan)
+        for test in _get_test_plan_list(test_plan_path):
+            test_list.append(os.path.join(test_plan_path, test))
+
+    if args.test_case:
+        for test in args.test_case:
+            test_case_path = os.path.join(args.testcase_path, test)
+            test_list.append(test_case_path)
```

Comment:
is this intentional to use both --test-plan and --test-case together? Code allows for that.

Reply:
Added to exclusive group.